docs/cli.md

```diff
@@ -456,10 +456,10 @@ gcli group delete --group new_group
 gcli tag details --tag tagA
 ```
 
-#### Create a tag
+#### Create tags
 
 ```bash
- gcli tag create --tag tagA
+ gcli tag create --tag tagA tagB
```

Comment:
Still, I don't agree to the syntax.

When you are creating a tag, you specify the tags as the operand. I.e

`gcli tag create tagA` or `gcli tag create tagA,tagB`

It is very uncommon to have `--tag` as the argument when you invoke `tag create`.